src/goto-harness/recursive_initialization.cpp

```diff
@@ -155,7 +155,10 @@ void recursive_initializationt::initialize_struct_tag(
   auto const &ns = namespacet{goto_model.symbol_table};
   for(auto const &component : ns.follow_tag(type).components())
   {
-    initialize(member_exprt{lhs, component}, depth, new_known_tags, body);
+    if(!component.get_is_padding())
+    {
+      initialize(member_exprt{lhs, component}, depth, new_known_tags, body);
+    }
```

Comment:
What about function-pointer members - are they correctly supported at the moment? Else those may need a special case as well.